source/standalone/tutorials/04_sensors/benchmark_cameras.py

```diff
@@ -0,0 +1,668 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+This script might help you determine how many cameras your system can realistically run
+at different desired settings. This script can also be used to sanity check visually
+what the camera images and or point clouds output from the replicator looks like.
+
+
+.. code-block:: bash
+
+    # Usage with GUI
+    ./isaaclab.sh -p source/standalone/tutorials/04_sensors/benchmark_cameras.py -h
+
+    # Usage with headless
+    ./isaaclab.sh -p source/standalone/tutorials/04_sensors/benchmark_cameras.py -h --headless
+
+"""
+
+"""Launch Isaac Sim Simulator first."""
+
+import argparse
+from collections.abc import Callable
+
+from omni.isaac.lab.app import AppLauncher
+
+# parse the arguments
+args_cli = argparse.Namespace()
+
+parser = argparse.ArgumentParser(description="This script can help you benchmark how many cameras you could run.")
+
+
+def add_cli_args(parser):
+    """Add the benchmarking CLI args"""
+    parser.add_argument(
+        "--visualize",
+        action="store_true",
+        default=False,
+        required=False,
+        help=(
+            "Whether to visualize. Only switch to True if you don't care about the benchmarking results"
+            " and are instead visually checking replicator output."
+        ),
+    )
+
+    parser.add_argument(
+        "--num_tiled_cameras",
+        type=int,
+        default=2,
+        required=False,
+        help="How many tiled cameras to create",
+    )
+
+    parser.add_argument(
+        "--num_standard_cameras", type=int, default=1, required=False, help="How many normal cameras to create"
+    )
+
+    parser.add_argument(
+        "--num_ray_caster_cameras", type=int, default=1, required=False, help="How many normal cameras to create"
+    )
+
+    parser.add_argument(
+        "--tiled_camera_replicators",
+        nargs="+",
+        type=str,
+        default=["rgb", "depth"],
+        help="What replicators to use for the tiled camera",
+    )
+
+    parser.add_argument(
+        "--standard_camera_replicators",
+        nargs="+",
+        type=str,
+        default=["rgb", "distance_to_image_plane"],
+        help="What replicators to use for the usd camera",
```

Comment:
Please avoid questions as descriptions.

```suggestion
        help="The data types rendered by the usd camera",
```